python/paddle/tests/CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ set_tests_properties(test_vision_models PROPERTIES TIMEOUT 120)
 set_tests_properties(test_dataset_uci_housing PROPERTIES TIMEOUT 120)
 set_tests_properties(test_dataset_imdb PROPERTIES TIMEOUT 300)
 set_tests_properties(test_pretrained_model PROPERTIES TIMEOUT 600)
+set_tests_properties(test_callback_wandb PROPERTIES TIMEOUT 300)
```

Comment:
Since the elapsed time of `test_callback_wandb` are less than 30 seconds, thus, `TIMEOUT 300`->`TIMEOUT 60`?
```
2022-10-20 18:46:06     Test #463: test_callback_wandb ..............   Passed   28.08 sec
2022-10-20 18:46:06     Start 463: test_callback_wandb
2022-10-20 18:46:29     Test #463: test_callback_wandb ..............   Passed   23.44 sec
2022-10-20 18:46:29     Start 463: test_callback_wandb
2022-10-20 18:46:53 1/1 Test #463: test_callback_wandb ..............   Passed   23.59 sec
```

Reply:
I've made the requested change